src/main/java/lv/ctco/javaschool/auth/control/UserStore.java

```diff
@@ -75,4 +78,13 @@ void validatePassword(String password) throws InvalidPasswordException {
             throw new InvalidPasswordException();
         }
     }
+
+    public UserLoginDto convertToDto(User user) {
```

Comment:
Please extract DTO conversion functions into a separate class. `UserStore` has a specific functionality - interaction with DB, and DTO conversion is a different functionality.